_targets.R

```diff
@@ -64,13 +64,16 @@ targets_list <- list(
   # Get mapping of which cells are in which tiles (w/ no spatial info)
   tar_target(cell_tile_xwalk_df, get_cell_tile_xwalk(grid_cell_centroids_sf, grid_tiles_sf)),
 
-  # Get mapping of which lakes are in which cells (w/ no spatial info). Not currently
-  # being used, but this info saved as a file could be useful in the modeling part so
-  # that we know what data to pull for each lake).
-  tar_target(lake_cell_xwalk_df, get_lake_cell_xwalk(query_lake_centroids_sf, grid_cells_sf)),
-  tar_target(lake_cell_xwalk_csv, {
-    out_file <- "7_drivers_munge/out/lake_cell_xwalk.csv"
-    write_csv(lake_cell_xwalk_df, out_file)
+  # Get mapping of which lakes are in which cells and tiles (w/ no spatial info). Not currently
+  # being used here, except for plotting, but is being used in `lake-temperature-process-models`
+  # so that we know what meteorological data to pull for each lake.
+  tar_target(lake_cell_tile_xwalk_df,
+             get_lake_cell_tile_xwalk(query_lake_centroids_sf, grid_cells_sf, cell_tile_xwalk_df)),
+  tar_target(lake_cell_tile_xwalk_csv, {
+    out_file <- "7_drivers_munge/out/lake_cell_tile_xwalk.csv"
+    lake_cell_tile_xwalk_df %>%
+      select(site_id, state, cell_no, tile_no) %>%
```

Comment:
I know we talked about this but now that I am seeing it here, I wonder if the `select()` should just go inside of `get_lake_cell_tile_xwalk()`. I would expect the `_df` and the `_csv` with the same name to contain the same info.